test/functional/p2p_blockfilters.py

```diff
@@ -13,6 +13,7 @@
     hash256,
     msg_getcfcheckpt,
     msg_getcfheaders,
+    msg_getcfilters,
```

Comment:
nit-ish: the comment up top in `p2p_blockfilters.py` is still missing `cfilters`.

Reply:
Let's fix this up in a follow-up